src/tests/class_tests/openms/source/IsotopeLabelingMDVs_test.cpp

```diff
@@ -283,20 +283,20 @@ START_SECTION(( void IsotopeLabelingMDVs::calculateMDVs(
   lactate_normsum.push_back(lactate_1_normsum);
   lactate_normsum.push_back(lactate_2_normsum);
 
-//  isotopelabelingmdvs.calculateMDVs(lactate_normmax, lactate_normalized_normmax, IsotopeLabelingMDVs::MassIntensityType::NORM_MAX, IsotopeLabelingMDVs::FeatureName::PEAK_APEX_INT);
-//  isotopelabelingmdvs.calculateMDVs(lactate_normsum, lactate_normalized_normsum, IsotopeLabelingMDVs::MassIntensityType::NORM_SUM, IsotopeLabelingMDVs::FeatureName::PEAK_APEX_INT);
+  isotopelabelingmdvs.calculateMDVs(lactate_normmax, lactate_normalized_normmax, IsotopeLabelingMDVs::MassIntensityType::NORM_MAX, "peak_apex_int");
+  isotopelabelingmdvs.calculateMDVs(lactate_normsum, lactate_normalized_normsum, IsotopeLabelingMDVs::MassIntensityType::NORM_SUM, "peak_apex_int");
 
   for(size_t i = 0; i < lactate_normalized_normmax.size(); ++i)
   {
     for(size_t j = 0; j < lactate_normalized_normmax.at(i).getSubordinates().size(); ++j)
     {
       if (i == 0) // lactate_1
       {
-        TEST_REAL_SIMILAR(lactate_normalized_normmax.at(i).getSubordinates().at(j).getIntensity(), L1_norm_max.at(j));
+        TEST_REAL_SIMILAR(lactate_normalized_normmax.at(i).getSubordinates().at(j).getMetaValue("peak_apex_int"), L1_norm_max.at(j));
```

Comment:
are all changes related to peak_apex_int intended?

Reply:
yes because updating a feature with a peak_apex_int value doesn't translate to the intensity and vice versa